### ISSUE TITLE
API password requirement

### ISSUE THREAD

**Issue by InfiniteImp** [bzminer/bzminer#203]
First off, the HTTP API is amazing!  Love the way it was implemented.  Best I've seen in any miner.  I enabled it on one of my rigs (specified parms http_address, http_port and http_password in the batch file).   When I used Chrome to go to that IP/port, it automatically pulled up all my rigs and allowed me to view their stats and console output.  I didn't even need to edit their batch files to specify the above parms.   

That's also why I'm creating this issue.   Thought you'd want to know that all the rigs were accessible and automatically shown even though they did not have a password set in their batch files.  I'm totally good with this, I prefer it, but figured you'd want to know as the readme file says that calling bzminer without a password would disable the API.  Oh ya, I should also add that it didn't ask me for the password at any time :).  Just putting this out there in case it's not working as intended.

**Comment by bzminer** [comment]
hey @InfiniteImp ! i'm glad you like it! there's a lot more i plan to do with it, just time has been limited and i haven't been able to get back to things like adding back in the real-time management (like choosing mining algos, pausing/starting gpus, and oc's), which leads to your question about the password

So, the api does currently allow you to view the stats without a password, however "update" commands (which the api does support but i have not added back into the gui yet) will require the password.

This is actually the second comment i've gotten about viewing the stats without a password, so my plan has been to add an option to require the password for even "viewing" stats. i can make this a priority if you'd like and get into the next release

I appreciate you bringing this up to me. I had assumed people wouldn't mind viewing by default but i'm more than happy to require a password for even viewing~

If you find any other issues or things you'd like to see in the UI please let me know!

**Comment by InfiniteImp** [comment]
No, don't change it.  The fewer batch files I need to modify the better!  
Your assumption was correct.

The only improvement I can think of so far (it's only been 6 hours) is the console part of the window.  Is there a setting to have it display the most recent data in real-time?  Right now the cursor stays where it was last and new data appears below off-screen.  I need to hit the "scroll to bottom" button to get the most recent info on the screen.  Can that be set as automatic?

Thanks.

**Comment by InfiniteImp** [comment]
Oh ya, one other thing.  In the "Pools" view, below the "radiant info (the algo I'm mining) there is a section for "rvn".  I assume that's Ravencoin but I'm not mining that.  I've also seen (I don't recall which view) "zil" shown. I'm also not mining that.  

Thoughts?
